mono/metadata/class-internals.h

```diff
@@ -755,17 +755,17 @@ typedef struct {
 
 typedef struct {
 	guint64 new_object_count;
-	size_t initialized_class_count;
-	size_t generic_vtable_count;
+	gsize initialized_class_count;
```

Comment:
I'm just working with the code since June and most things I do are based on observation. Personally, I do not not have any preferences (yet). However, I understand that the glib types are preferred wherever possible. (I might be wrong there!)

My thought was that atomic.h uses `gint*` so it might be nice to use all `g*` types in racing.h to keep some kind of consistency. And, since `racing.h` works with `gsize` AND `mono_stats` has this `guint64` value anyways, I thought that (again talking about consistency) it would be nice to have those affected fields defined as `gsize` in `mono_stats` as well.